### ISSUE TITLE
Sourcemaps on the server broken in 1.x

### ISSUE THREAD

**Issue by jbbr** [thereactivestack/meteor-webpack#84]
Sourcemaps seems to be broken for the server in 1.x release (tested 1.0.8).

What works:
- source maps in client side code (simple eval-style maps)

What doesn't work:
- source maps in Meteor methods (both client simulation and server execution)
- source maps anywhere in server code

I tried with both Meteor 1.2.1 and 1.3beta12 using kickstart-meteor-react-router and a project of mine.

Would be nice to have working source maps on the server again.

Example trace of an intentionally error in `TodoApps/server/publications.js` (Just added `this.is.an.error;` to top level)

```
W20160309-15:27:14.874(1)? (STDERR) TypeError: Cannot read property 'is' of undefined
W20160309-15:27:14.874(1)? (STDERR)     at Object.eval (eval at <anonymous> (/path/to/kickstart-meteor-react-router/.meteor/local/build/programs/server/app/server.js:69:2), <anonymous>:11:10)
W20160309-15:27:14.874(1)? (STDERR)     at Object.module.exports (/path/to/kickstart-meteor-react-router/.meteor/local/build/programs/server/app/server.js:69:2)
W20160309-15:27:14.874(1)? (STDERR)     at __webpack_require__ (/path/to/kickstart-meteor-react-router/.meteor/local/build/programs/server/app/server.js:24:30)
W20160309-15:27:14.874(1)? (STDERR)     at Object.eval (eval at <anonymous> (/path/to/kickstart-meteor-react-router/.meteor/local/build/programs/server/app/server.js:63:2), <anonymous>:3:1)
W20160309-15:27:14.875(1)? (STDERR)     at Object.module.exports (/path/to/kickstart-meteor-react-router/.meteor/local/build/programs/server/app/server.js:63:2)
W20160309-15:27:14.875(1)? (STDERR)     at __webpack_require__ (/path/to/kickstart-meteor-react-router/.meteor/local/build/programs/server/app/server.js:24:30)
W20160309-15:27:14.875(1)? (STDERR)     at Object.eval (eval at <anonymous> (/path/to/kickstart-meteor-react-router/.meteor/local/build/programs/server/app/server.js:51:2), <anonymous>:5:1)
W20160309-15:27:14.875(1)? (STDERR)     at Object.module.exports (/path/to/kickstart-meteor-react-router/.meteor/local/build/programs/server/app/server.js:51:2)
W20160309-15:27:14.875(1)? (STDERR)     at __webpack_require__ (/path/to/kickstart-meteor-react-router/.meteor/local/build/programs/server/app/server.js:24:30)
W20160309-15:27:14.876(1)? (STDERR)     at module.exports (/path/to/kickstart-meteor-react-router/.meteor/local/build/programs/server/app/server.js:44:18)
=> Exited with code: 8
```

**Comment by eXon** [comment]
I agree source-map is a must. I'll look at how we can fix this.

**Comment by eXon** [comment]
This is fixed in 1.0.9
